### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by grempe**
* Use 'randombytes' browserify package instead of raw 'crypto' which results in no change in functionality but a massive reduction in the size of the browserify output since the entire node crypto lib is no longer being included.
* Added a simple index.html to allow quick clone and local running of niceware to generate passphrase safely and locally. For example on macOS simply do 'open index.html' in the cloned repo to run it in your browser.
* Added a little text to the README about wordlist length and bits of entropy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ A module for converting cryptographic keys into human-readable phrases, similar
 to [Diceware](http://world.std.com/~reinhold/diceware.html) but with a larger
 wordlist. With Niceware, a 128-bit key is equivalent to an 8-word phrase.
 
+The wordlist contains 65,536 English words, which results in
```
Comment (diracdeltas) [inline README.md]: Thanks! I think this first part is redundant given the change in 57568c024adca02a6c73629f1530f2b059ebf1db.

Comment (grempe) [reply README.md]: Sure. Modify as you see fit.

File: package.json [hunk 2]
```diff
@@ -42,6 +42,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "binary-search": "^1.3.2"
+    "binary-search": "^1.3.2",
+    "randombytes": "^2.0.3"
```
Comment (diracdeltas) [inline package.json:46]: Great idea, i think the browser package size reduction is definitely worth the extra dependency in this case.

File: README.md (hunk 1 above)
Comment (diracdeltas) [reply README.md]: @grempe please take out this change, which will probably resolve the merge conflict, and then i can merge

File: package.json (hunk 2 above)
Comment (grempe) [reply package.json:46]: Done, I replaced the HEAD commit on my branch with a new one 1a9d51f that has no README change, only the index.html.

**Comment by diracdeltas** [comment]
thanks!

**Comment by grempe** [comment]
Awesome. Glad it helped.

**Pull request merged**